docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,466 @@
+---
+title: Getting started with KLT
+description: Get started with the Keptn Lifecycle Toolkit
+weight: 10
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+Keptn Lifecycle Toolkit works whether or not you use a GitOps strategy.
+The following is an imperative walkthrough.
+
+## Prerequisites
+
+- A Kubernetes cluster >= 1.24 (we recommend [Kubernetes kind](https://kind.sigs.k8s.io/docs/user/quick-start/))
+  (`kind create cluster`)
+- [Helm](https://helm.io) CLI available
+
+## Objectives
+
+- Install Keptn Lifecycle Toolkit on your cluster
+- Annotate a namespace and deployment to enable Keptn Lifecycle Toolkit
+- View DORA Metrics
+- Install Grafana and Observability tooling to view DORA metrics
+
+## System Overview
+
+By the end of this page, here is what will be built.
+This system will be built in stages.
+
+![system overview](/docs/install/assets/install01.png)
+
+## The Basics: A Deployment, Keptn and DORA Metrics
```

Comment:
Should this be entitled "Implement observability" or something?  Breaking this up into the three "use cases" might make it more digestible and also match the direction we're going with the product.

Reply:
I don't think so. That's fine for people familiar with the project or the "marketecture" of Keptn but it would mean little to a first time reader.